projectgenerator/tests/testdata/ilimodels/ExceptionalLoadsRoute_V1.ili

```diff
@@ -10,7 +10,7 @@ VERSION "2017-02-08"  =
 
   TOPIC TypeOfRouteCatalogue
   EXTENDS CatalogueObjects_V1.Catalogues =
-
+    !!@ dispExpression="t_id||type"

```

Comment:
While technically possible, it's probably better to use something like `COALESCE(Type, ' (', RoadWidth, ' m)')` to promote good practice (Especially the `t_id` is meaningless except for the database backend).

Reply:
Okay. Do we use this file for other things than tests?

Reply:
nope, not actually.